app/models/miq_filter.rb

```diff
@@ -43,28 +43,43 @@ def self.belongsto2object_list(tag)
     # /belongsto/ExtManagementSystem|<name>/EmsCluster|<name>/EmsFolder|<name>
     raise _("invalid tag: %{tag}") % {:tag => tag} unless tag.starts_with?("/belongsto/ExtManagementSystem")
     parts = tag.split("/")[2..-1]
+    depth = parts.size - 1 # ancestry uses 0 based depth
 
-    # root object
-    klass, name = parts.shift.split("|")
-    obj = find_object_by_name(klass, name)
+    # Get the root EMS object
+    # TODO: For tree queries deeper than 1, we don't actually need the ems object,
+    #       so find a way to just get the id
```

Comment:
yeah that's why I'm punting for now - in a follow up PR I'm hoping to deal with multiple belongsto strings simultaneously, and I think this problem might go away, since I will just get the object from a longer string that's a super-set